docs/blog/2019-03-01-localization-with-gatsby-and-sanity/index.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Localization with Gatsby and Sanity.io
+date: 2019-03-01
+author: Travis Stanley
+tags: ["sanity", "localization", "l10n"]
+image: "./images/gatsby-sanity-l10n.png"
+showImageInArticle: true
+---
+
+Localization is a common problem and there are _many_ solutions, each with their own benifits and trade-offs. With Gatsby and Sanity.io its possible to achieve a solution that is easy to work with and extend. If you haven't already read about why Gatsby and Sanity.io pair so well check out the blog post by [Knut Melvær](/contributors/knut-melvaer/): [Blazing fast development with Gatsby and Sanity.io](/blog/2019-01-25-blazing-fast-development-with-gatsby-and-sanity-io).
```

Comment:
How are these two blog posts different? Is this one more of a how-to (perhaps that could go into the title)?

Reply:
This one is specifically a how-to for localization with Sanity and Gatsby. I hesitate to change the title only because there are probably numerous ways to achieve the same effect. "How to localize with Gatsby and Sanity" puts a lot of force behind it...